thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/ThirdEyeAnomalyConfiguration.java

```diff
@@ -26,6 +29,32 @@
   private TaskDriverConfiguration taskDriverConfiguration = new TaskDriverConfiguration();
   private String failureFromAddress;
   private String failureToAddress;
+  private String keyPath;
+  private List<String> calendars;
```

Comment:
Initialize this list with Collections.emptyList() to prevent nullPointException in the future.